libpod/util.go

```diff
@@ -35,10 +34,10 @@ func StringInSlice(s string, sl []string) bool {
 	return false
 }
 
-// FuncTimer helps measure the execution time of a function
-// For debug purposes, do not leave in code
-// used like defer FuncTimer("foo")
-func FuncTimer(funcName string) {
-	elapsed := time.Since(time.Now())
-	fmt.Printf("%s executed in %d ms\n", funcName, elapsed)
+// HasTransport determines if the image string contains '://', returns bool
```

Comment:
Why are you removing FuncTimer?

Reply:
Apparently my merge dance wasn't as smooth as it should have been.  I didn't men to remove that, back to dancing.